features/shared/checkout_previous_branch_after_success/same_current_branch/previous_branch_same.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Allow checking out the correct previous Git branch after running a Git Town commmand that leaves the current and previous branches intact
```

Comment:
Feature: Git Town commands that don't change the current and previous branch preserve Git history